src/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts

```diff
@@ -22,12 +21,10 @@ export function getWorksForSaleRouteVariables({ artistID }, { location }) {
     "ARTIST_SERIES",
   ]
 
-  const includeBlurHash = !!getFeatureFlag("diamond_blurhash-on-artist-pages")
-
   return {
     input: filterParams,
     aggregations,
     artistID,
-    includeBlurHash,
+    includeBlurHash: false,
```

Comment:
Rather than remove all of the query arguments scattered around. Dunno if there's a preferable way to do this ATM.